ocaml/src/cli_asd.ml

```diff
@@ -582,11 +588,14 @@ let asd_statistics hosts port_o transport asd_id
                 | Some (claim_info, osd) ->
                   let conn_info =
                     let open Nsm_model.OsdInfo in
-                    let conn_info' = get_conn_info osd.kind in
+                    let conn_info' = match osd.kind with
+                      | Asd (x, _) -> x
+                      | Kinetic _ | Alba _ -> assert false
```

Comment:
maybe a function to get the Asd from the osd's kind ? (It's not the first time I see this inline extraction)